script_facade/models/r1/patient.py

```diff
@@ -0,0 +1,85 @@
+
+
+class Patient(object):
+
+    def __init__(self):
+        self.given_name = None
+        self.family_name = None
+        self.gender = None
+        self.birthdate = None
+        self.address = None
+
+    @classmethod
+    def from_xml(cls, xml_element):
```

Comment:
lot of similar code in here.  would it be worth a nested function to take a path and return a parsed value or None?

Reply:
That makes a lot of sense. 

When I cleanup the XPath expressions (to use the correct namespace removing the ugly `local-name()=`) I'll add a helper function to do exactly that.